draft-ietf-scitt-software-use-cases.md

```diff
@@ -298,35 +297,37 @@ A consumer of a released software wants:
+Through this connection, the telemetry data and analytics are transfered, and the device receives firmware updates published by vendors.
+During initialization, general purpose computers can also have resource constraints like that of constrained IoT devices. 
+Verification of hardened configuration of the computer's chipset for ongoing telemetry is increasingly important. 
+After initialization, even if not constrained similarly to IoT devices, the computer's operating system can facilitate telemetry about telemetry settings and measure differences at scale.
 The public network, open distribution system, and firmware update process create several security challenges.
 
-Consumers and other interested parties of a firmware update ecosystem wants:
+Consumers and other interested parties of a firmware update ecosystem want to:
 
-- to know that the received firmware for system update is not faulty or malicious
-- to know if the signing identity used to assert the authenticity of the firmware is somehow used to sign unintended updates
-- to ascertain that the released firmware is not subverted or compromised due to an insider risk - be it malicious or otherwise
-- to confirm that the publishers know if their deliverable has been compromised. (For example, can they trust their key protection or audit logging?)
-- to know how the update client on an instance of a health monitoring system discerns a general update from one specially crafted for just a small subset of a fleet of devices
-- to know the firwmare has effectively maintained or changed applicable hardware settings once firmware is applied
+- know that the received firmware for system update is not faulty or malicious
+- know if the signing identity used to assert the authenticity of the firmware is somehow used to sign unintended updates
+- ascertain that the released firmware is not subverted or compromised due to an insider risk - be it malicious or otherwise
+- confirm that the publishers knows if their deliverable has been compromised. For example, can they trust their key protection or audit logging?
+- know how the update client on an instance of a health monitoring system discerns a general update from one specially crafted for just a small subset of a fleet of devices
+- know if the firwmare has effectively maintained or changed applicable hardware settings after installation
 
 There is no standardized way to:
 
-- provide an update framework that allows validation of authenticity of firmware revisions
+- provide an update framework that allows validation of authenticity of firmware revisions (except for TUF and SUIT, fixme)
```

Comment:
what is the `fixme` reference?

Reply:
attempted a fix in commit 5865af8759e5a8fc45e1afb0b57247efea03b483